index.html

```diff
@@ -29,7 +30,8 @@ <h1 id="title" class="text-center">freeCodeCamp Survey Form</h1>
             required
           />
         </div>
-        <div class="form-group">
+        <div class="Form-group">
+
```

Comment:
Kindly do not use an unnecessary blank line.

Reply:
- [ ] It is great that you have told the student very politely to make the following change. :+1: But you didn't give any reason for the requested change. Please add a reason for the requested change :+1:

Reply:
Kindly refrain from using an unnecessary blank line as blank lines, spaces, or indentations are not used without a reason.

Reply:
Great readability of your html makes it much easier to maintain in the future, as you'll be able to find elements quicker. Also, you'll never know who might need to look at your code later on.